data/mods/ssb/moves.ts

```diff
@@ -862,6 +862,31 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		type: "Psychic",
 	},
 
+	// Ransei
+	ripsei: {
+		num: 515,
+		accuracy: 100,
+		basePower: 0,
+		damageCallback(pokemon) {
+			const damage = pokemon.hp;
+			const target = pokemon.side.foe.active[0];
+			if (target && target.hp <= 0) target.faint();
```

Comment:
this feels rather hacky, I would rather have the same effects from final gambit, and then have a check for fainting in an onAfterMove function (see: cant say's ability)

Reply:
also if there was a sturdy mon, they would be cheated out of their 1 hp.

Reply:
turns out this isnt gonna work, im unclaiming this one